pkgs/applications/misc/prusa-slicer/default.nix

```diff
@@ -161,16 +156,16 @@ stdenv.mkDerivation (finalAttrs: {
     # dlopen(3) for context.
     if [ -f "src/libslic3r/Format/STEP.cpp" ]; then
       substituteInPlace src/libslic3r/Format/STEP.cpp \
-        --replace 'libpath /= "OCCTWrapper.so";' 'libpath = "OCCTWrapper.so";'
+        --replace-fail 'libpath /= "OCCTWrapper.so";' 'libpath = "OCCTWrapper.so";'
     fi
     # https://github.com/prusa3d/PrusaSlicer/issues/9581
     if [ -f "cmake/modules/FindEXPAT.cmake" ]; then
       rm cmake/modules/FindEXPAT.cmake
     fi
 
     # Fix resources folder location on macOS
-    substituteInPlace src/PrusaSlicer.cpp \
-      --replace "#ifdef __APPLE__" "#if 0"
+    substituteInPlace src/CLI/Setup.cpp \
```

Comment:
This fails for super-slicer.

Reply:
oh god super-slicer.... Yeah, that is a bit broken.... Fixing that needs some work...

Reply:
Actually, it was only that, fixed.